rehfeldmedical/openshift-centos-playbooks
# SCAUT-365: Convert docker-ce playbook to re-usable role.

This PR converts the docker playbook into a re-usable docker-ce role. The role will uninstall any existing docker installation, then install docker-ce, preserving underlying docker storage.

Changes
====
1. Converted docker playbook into docker-ce role.
2. Made docker playbook use the new role.

Testing
====
1. Create a new VM via `vagrant up` in the vagrant playground with a `hosts.local` file set.
2. Login to VM, confirm IP address and add to `ansible.cfg`.
3. Confirm existing docker installation works and is 1.13.x
4. Run `./local-play.sh ../playbooks/docker.yml`
5. Confirm docker installation works and is 18.x docker-ce.
